rules/windows/command_and_control_sunburst_c2_activity_detected.toml

```diff
@@ -0,0 +1,53 @@
+[metadata]
+creation_date = "2020/12/14"
+maturity = "production"
+updated_date = "2020/12/14"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects post-exploitation command and control activity of the SUNBURST backdoor. This rule requires Sysmon for
+Winlogbeats.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "SUNBURST Command and Control Activity"
```

Comment:
```suggestion
name = "SolarWinds Command and Control Activity"
```

Reply:
We wanted to focus on the malware vs. the vendor where possible.